shared/AppInsightsCommon/src/Enums.ts

```diff
@@ -47,3 +47,8 @@ export const DistributedTracingModes = createEnumStyle<typeof eDistributedTracin
     W3C: eDistributedTracingModes.W3C
 });
 export type DistributedTracingModes = number | eDistributedTracingModes;
+
+export const enum IThrottleMsgKey {
```

Comment:
Do these need to be strings?

Ideally, we want const enum's to be numeric as this produces smaller code.